addons/mail/static/src/components/file_uploader/file_uploader.js

```diff
@@ -91,11 +78,17 @@ class FileUploader extends Component {
      */
     _createTemporaryAttachments(files) {
         for (const file of files) {
-            this._createAttachment({
-                filename: file.name,
-                isTemporary: true,
-                name: file.name
-            });
+            this.env.models['mail.attachment'].create(
+                Object.assign(
+                    {},
+                    {
+                        filename: file.name,
+                        isTemporary: true,
+                        name: file.name
+                    },
```

Comment:
Don't need for an empty first object when the second object is created manually too